### PR TITLE
Added British words

### PR DESCRIPTION
**Pull request by Paradact**
Added two British words

### CODE REVIEW

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ analysed->analyzed
 analyser->analyzer
 analysers->analyzers
 analysing->analyzing
+armour->armor
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:13]: Add `armoured` and `armouring`?

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 2]
```diff
@@ -73,6 +74,7 @@ digitise->digitize
 digitised->digitized
 digitises->digitizes
 digitising->digitizing
+doughnut->donut
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:77]: Add plural `doughnuts`?

Comment (Paradact) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:79]: I've added these, thanks for your suggestions.

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 3]
```diff
@@ -10,6 +10,9 @@ analysed->analyzed
 analyser->analyzer
 analysers->analyzers
 analysing->analyzing
+armour->armor
+armoured->armored
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:14]: I am not a native speaker, but I would really add `armouring->armoring` here.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
```
E   error doughnut: correction 'donut' should be in aspell dictionaries (en_US) for dictionary /home/runner/work/codespell/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
```
Indeed, `donut` should be at least in the `en_US-large` _aspell_ dictionary, but it is not:

en_US-large

How do we work around this?

**Comment by DimitriPapadopoulos** [comment]
@Pradact Leaving out `donut` for now. I have created issue #2657 instead.

**Pull request merged**